### PR TITLE
Remove Duplicate Icons

### LINKED ISSUE #8556: Remove Duplicate Icons
## Feature Description

This spring, we implemented a new design system. Most icons were swapped to use that source, there's a few outliers that exist in the dashboard that should be retired and replaced with their twins in the design system. They are: 

- Edit icon

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

Removing the versions of these icons that exist in the dashboard and using the ones in the design system instead.

### PR DESCRIPTION
**Pull request by g-sqiao**
## Context

Replaced the original pencil icon with the one in design-system.

### QA

This is purely cosmetic. 

Go to Dashboard Settings and hover or focus on a publisher logo, the icon should be updated and the button is a little bigger, like this: 
https://user-images.githubusercontent.com/14279973/129640746-e5a28a68-b959-4727-86c9-179f94dea605.png

Please verify that clicking on the button still opens up the context menu as expected and that it looks like this screenshot.

### UAT

- [ ] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [ ] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [ ] I have tested this code to the best of my abilities
- [ ] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [ ] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [ ] I have added documentation where necessary
- [ ] I have added a matching `Type: XYZ` label to the PR

---

Fixes #

### CODE REVIEW

**Review by BrittanyIRL — CHANGES_REQUESTED** [review]
| This PR | Main | 
| --- | --- | 
| Screen Shot 2021-08-13 at 11 36 52 AM | Screen Shot 2021-08-13 at 11 37 08 AM | 

See the difference in size? 

So, with the SVGs to properly replace them you're going to have to adjust the size of a few things because the [SVG viewbox](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/viewBox) is different. You can see this if you open dev tools and inspect the edit icon in the button. 

Additionally, here the button isn't using the one in the design system and it probably could use the icon variant. 

Not to be pedantic, but screen shots of the changes are incredibly helpful. The PR description has a template built into it so that context and review are easier to glean. It would be useful to update that here. 

Thanks!

File: packages/dashboard/src/app/views/editorSettings/publisherLogo/popoverLogoContextMenu.js [hunk 1]
```diff
@@ -22,13 +22,13 @@ import { useCallback } from 'react';
...
 import {
   AnimatedContextMenu,
+  Icons,
   MenuItemProps,
 } from '@web-stories-wp/design-system';
 
 /**
  * Internal dependencies
  */
-import { EditPencil as EditPencilIcon } from '../../../../icons';
```
Comment (dreamofabear) [inline packages/dashboard/src/app/views/editorSettings/publisherLogo/popoverLogoContextMenu.js:28]: Should we also remove `EditPencil` from this `icons.js`?

File: packages/dashboard/src/icons/editPencil.svg [hunk 2]
```diff
@@ -1,3 +1,3 @@
-<svg fill="none" height="12" width="13" viewBox="0 0 13 12" width="13" xmlns="http://www.w3.org/2000/svg">
```
Comment (dreamofabear) [inline packages/dashboard/src/icons/editPencil.svg:1]: This file should be removed, no?

**Comment by dreamofabear** [comment]
> screen shots of the changes are incredibly helpful. The PR description has a template built into it so that context and review are easier to glean. It would be useful to update that here.

+1, ditto for the other PRs.

**Review by BrittanyIRL — COMMENTED** [review]
hi again! 

So i get that this is confusing and it was probably a kind of chaotic place to start 😂  I took some time and figured out how to swap this out. I'm going to lay it out here for you so that you can get more comfortable with the codebase. 


Open `dashboard/srcs/app/views/editorSettings/components.js`.
This is where all of the styled components live that make up the settings page. We're going to go look at the `LogoMenuButton` (should be about line 171).

Right off the bat, we can see that this "button" is not using the [`Button` component from the design system](https://google.github.io/web-stories-wp/storybook/?path=/story/designsystem-components-button--light-theme), so let's swap that out. We know we want it to be round (`BUTTON_VARIANTS.CIRCLE`) and based on looking at the rest of the page as well as the theme colors, we know that this button is the "wrong" (outdated) color. So let's make it a secondary button. You can also see that the "small" buttons are a smidge larger than these lil itty bitty ones, so let's ditch the custom styles in favor of continuity with the design system. You'll end up with a base that looks like this: 

```
export const LogoMenuButton = styled(Button).attrs({
  size: BUTTON_SIZES.SMALL,
  type: BUTTON_TYPES.SECONDARY,
  variant: BUTTON_VARIANTS.CIRCLE,
})
```

Now we're enabled to delete a bunch of lines that were making this custom button. You should be able to remove the following: 
```
  display: flex;
  align-items: center;
 width: 24px;
  height: 24px;
  text-align: center;
  padding: 0;
  color: ${({ theme }) => theme.colors.standard.white};
  background: ${({ theme }) => theme.colors.fg.secondary};
  border-radius: 50%;
  border: 1px solid transparent;
  cursor: pointer;
```

Now we have a pretty, consistent, round button. It should look like this: 
```
export const LogoMenuButton = styled(Button).attrs({
  size: BUTTON_SIZES.SMALL,
  type: BUTTON_TYPES.SECONDARY,
  variant: BUTTON_VARIANTS.CIRCLE,
})`
  opacity: ${({ isActive, menuOpen }) => (menuOpen || isActive ? 1 : 0)};
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
`
```

Next, let's do the icon. First, go delete the editPencil icon that you edited in `packages/dashboard/src/icons/editPencil.svg` 😅 ! Now, go to `editorSettings/publisherLogoContextMenu.js` and import `Icons` from the design-system (already listed at the top of external imports) and replace the `<EditPencilIcon />` with `<Icons.Pencil />`.

Go back to the `LogoMenuButton` styles and let's look at the svg styles getting applied. So, the old icon needed a bunch of stuff applied to it because of how the size was of the svg. The viewbox and the width were making it a little lopesided, super fun when you're trying to center something.  For comparison, the SVG we're deleting was ` height="12" width="13" viewBox="0 0 13 12" width="13"` and the new one is `width="32" height="32" viewBox="0 0 32 32"`.

The great news here is that we need **none** of that customization. Just delete it. If you're curious. Open your dev tools and inspect the SVG, see the styles applied to it and remove the styles by clicking the checkbox next to the styles one by one and see it get better. 

You should be able to end up with this code for the entire button: 
```
export const LogoMenuButton = styled(Button).attrs({
  size: BUTTON_SIZES.SMALL,
  type: BUTTON_TYPES.SECONDARY,
  variant: BUTTON_VARIANTS.CIRCLE,
})`
  opacity: ${({ isActive, menuOpen }) => (menuOpen || isActive ? 1 : 0)};
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
`;
```

| Code above | Staging | 
| --- | --- | 
|  Screen Shot 2021-08-16 at 3 18 52 PM |  Screen Shot 2021-08-16 at 3 05 10 PM | 


Does this make sense?

**Comment by g-sqiao** [comment]
> hi again!
> 
> So i get that this is confusing and it was probably a kind of chaotic place to start 😂 I took some time and figured out how to swap this out. I'm going to lay it out here for you so that you can get more comfortable with the codebase.
> 
> Open `dashboard/srcs/app/views/editorSettings/components.js`.
> This is where all of the styled components live that make up the settings page. We're going to go look at the `LogoMenuButton` (should be about line 171).
> 
> Right off the bat, we can see that this "button" is not using the [`Button` component from the design system](https://google.github.io/web-stories-wp/storybook/?path=/story/designsystem-components-button--light-theme), so let's swap that out. We know we want it to be round (`BUTTON_VARIANTS.CIRCLE`) and based on looking at the rest of the page as well as the theme colors, we know that this button is the "wrong" (outdated) color. So let's make it a secondary button. You can also see that the "small" buttons are a smidge larger than these lil itty bitty ones, so let's ditch the custom styles in favor of continuity with the design system. You'll end up with a base that looks like this:
> 
> ```
> export const LogoMenuButton = styled(Button).attrs({
>   size: BUTTON_SIZES.SMALL,
>   type: BUTTON_TYPES.SECONDARY,
>   variant: BUTTON_VARIANTS.CIRCLE,
> })
> ```
> 
> Now we're enabled to delete a bunch of lines that were making this custom button. You should be able to remove the following:
> 
> ```
>   display: flex;
>   align-items: center;
>  width: 24px;
>   height: 24px;
>   text-align: center;
>   padding: 0;
>   color: ${({ theme }) => theme.colors.standard.white};
>   background: ${({ theme }) => theme.colors.fg.secondary};
>   border-radius: 50%;
>   border: 1px solid transparent;
>   cursor: pointer;
> ```
> 
> Now we have a pretty, consistent, round button. It should look like this:
> 
> ```
> export const LogoMenuButton = styled(Button).attrs({
>   size: BUTTON_SIZES.SMALL,
>   type: BUTTON_TYPES.SECONDARY,
>   variant: BUTTON_VARIANTS.CIRCLE,
> })`
>   opacity: ${({ isActive, menuOpen }) => (menuOpen || isActive ? 1 : 0)};
>   position: absolute;
>   top: 50%;
>   left: 50%;
>   transform: translate(-50%, -50%);
> `
> ```
> 
> Next, let's do the icon. First, go delete the editPencil icon that you edited in `packages/dashboard/src/icons/editPencil.svg` 😅 ! Now, go to `editorSettings/publisherLogoContextMenu.js` and import `Icons` from the design-system (already listed at the top of external imports) and replace the `<EditPencilIcon />` with `<Icons.Pencil />`.
> 
> Go back to the `LogoMenuButton` styles and let's look at the svg styles getting applied. So, the old icon needed a bunch of stuff applied to it because of how the size was of the svg. The viewbox and the width were making it a little lopesided, super fun when you're trying to center something. For comparison, the SVG we're deleting was ` height="12" width="13" viewBox="0 0 13 12" width="13"` and the new one is `width="32" height="32" viewBox="0 0 32 32"`.
> 
> The great news here is that we need **none** of that customization. Just delete it. If you're curious. Open your dev tools and inspect the SVG, see the styles applied to it and remove the styles by clicking the checkbox next to the styles one by one and see it get better.
> 
> You should be able to end up with this code for the entire button:
> 
> ```
> export const LogoMenuButton = styled(Button).attrs({
>   size: BUTTON_SIZES.SMALL,
>   type: BUTTON_TYPES.SECONDARY,
>   variant: BUTTON_VARIANTS.CIRCLE,
> })`
>   opacity: ${({ isActive, menuOpen }) => (menuOpen || isActive ? 1 : 0)};
>   position: absolute;
>   top: 50%;
>   left: 50%;
>   transform: translate(-50%, -50%);
> `;
> ```
> 
> Code above	Staging
> Screen Shot 2021-08-16 at 3 18 52 PM	Screen Shot 2021-08-16 at 3 05 10 PM
> Does this make sense?

OMG! You are AMAZING, Brittany!! <3 Thank you so much for the detailed explanations! LOL
Yup, it makes sense! Let me do the change and send it to you!

**Comment by g-sqiao** [comment]
| After | Before | 
| --- | --- | 
|  |  |

**Review by BrittanyIRL — APPROVED** [review]
👏 👏 👏  
Great job, thanks for stickin' with it.

I'm re-requesting @choumx for a secondary review and then this will go to QA.

**Review by dreamofabear — APPROVED** [review]

**Comment by swissspidy** [comment]
Great work! This now just needs some QA instructions for the affected areas to prevent regressions.

**Comment by swissspidy** [comment]
@BrittanyIRL could you perhaps help with QA instructions here?

**Comment by BrittanyIRL** [comment]
(I see the conflict and will fix first thing in AM)

**Pull request merged**